tests/test2-cli-init.test.ts

```diff
@@ -9,12 +9,17 @@ test("basic init", async () => {
   const { stdout, stderr } = await runCommand("tsci init")
   expect(stderr).toBe("")
 
-  // List directory contents
-  const dirContents = await readdir(tmpDir)
+  const dirContents = (await readdir(tmpDir)).sort()
```

Comment:
you can do `.join("\n,")` to maybe fix that consistency issue, although i've never seen an issue with `.sort()`